### PR TITLE
feat: add Aplib to Unity

### PR DESCRIPTION
**Pull request by joachimdekker**
# Description

This PR proposes to add the testrunner for testing frameworks such as Unity Test Framework to use.

# Testability

The testrunner literally tests the game. You could try to run this version in Unity, but installing that is quite the hassle.

### CODE REVIEW

**Review by SilasPeters — APPROVED** [review]

File: aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs
Comment (JensSteenmetz) [inline aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs:1]: Keep this file or not? I thought we did that in a separate PR.

File: aplib.net-demo/Assets/Scenes/FirstTestScene.unity
Comment (JensSteenmetz) [inline aplib.net-demo/Assets/Scenes/FirstTestScene.unity:1]: The test scenes are supposed to be in the different PR as well I suppose

Comment (JensSteenmetz) [reply aplib.net-demo/Assets/Scenes/FirstTestScene.unity:1]: The same applies to some more Unity-related files.

File: aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs
Comment (joachimdekker) [reply aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs:1]: I'd say this is to prove that Aplib works!

File: aplib.net-demo/Assets/Scenes/FirstTestScene.unity
Comment (joachimdekker) [reply aplib.net-demo/Assets/Scenes/FirstTestScene.unity:1]: I will remove the scenes that have nothing to do with this PR!

File: aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs
Comment (JensSteenmetz) [reply aplib.net-demo/Assets/Testing/AplibTests/MyFirstAplibTest.cs:1]: Good point!

**Review by JensSteenmetz — APPROVED** [review]
LGTM!

**Review by Tboefijn — APPROVED** [review]
